containers/pre_built_workers/cxx/run_worker.sh

```diff
@@ -0,0 +1,23 @@
+# Copyright 2020 gRPC authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -ex
+
+cd /source/code
+
+pip3 install six
+
+python3 /source/code/tools/run_tests/start_port_server.py
```

Comment:
nit: why do you need the port server?

Reply:
The original reason was: when I tried to run the example load test, the error message direct me to run https://github.com/grpc/grpc/blob/master/tools/run_tests/start_port_server.py, that is only reason behinds this script. After @Ben found out how to feed the server port number directly to workers, the script is deleted and the port is feed directly to the server. Verified with an example load test, worker okay.